cmd/root.go

```diff
@@ -170,6 +171,9 @@ func init() {
 	// create two new flags, one for host and one for port
 	rootCmd.Flags().String("host", "localhost", "host to listen on")
 	rootCmd.Flags().String("port", "8042", "port to listen on")
+	rootCmd.Flags().String("brokers", "localhost:9092", "brokers uri")
```

Comment:
Docker compose in docs exposes 19092 (current hardcoded value) so changing to 9092 breaks some examples. 9092 is kafka default so I prefer that.

Docs should be revisited - I'm happy to run through them & resolve any issues if you don't want to address that in current PR.